src/scripts/docs/examples/DropdownMultiSelectAdvanced.example.txt

```diff
@@ -0,0 +1,82 @@
+class ComponentExample extends React.Component {
+  state = {
+    selectedItems: {},
+    selectedItemsIds: [],
+    itemSearchLoading: false,
+    items: {
+          1: { id: 1, title: 'Ben Bruning', name: 'BB', profileImageUrl: '' },
+          2: { id: 2, title: 'Blake Guilloud', name: 'BG', profileImageUrl: '' },
+          3: { id: 3, title: 'Craig Anthony', name: 'CA', profileImageUrl: '' },
+          4: { id: 4, title: 'Grady Rogers', name: 'GR', profileImageUrl: '' },
+          5: { id: 5, title: 'Keaton Foster', name: 'KF', profileImageUrl: '' },
+          6: { id: 6, title: 'Lauren Farr', name: 'LF', profileImageUrl: '' },
+          7: { id: 7, title: 'Keaton Foster',name: 'KF', profileImageUrl: '' },
+          8: { id: 8, title: 'Paul Griffin', name: 'PG', profileImageUrl: '' },
+          9: { id: 9, title: 'Rob Whelan', name: 'RW', profileImageUrl: '' },
+          10: { id: 10, title: 'Terry Kennair', name: 'TK', profileImageUrl: '' },
+        },    
+    itemsIds: [],
+    itemsCopy: {},    
+  }
+
+  componentWillMount(){
+    this.setState({ itemsIds: Object.keys(this.state.items), itemsCopy: this.state.items });
+  }
+
+  fetchAllItems = (searchText) => {
+  //This code is written for a case, If we are filtering from the store data only and not from api, Otherwise we will only call api
```

Comment:
Please move this comment above the function declaration. It says it will call API, but I'm not seeing where that is done. Consider making this comment more clear.

Reply:
@alexgsummer Ok I will make it more clear in code also, From this comment I want to say that, I am not making any api call in this example, I am filtering the data from the stored state, but in case of members and  contacts filter of audit log, I will be calling api. So, this code can be used for both cases, but in the example i have used only one case.